### ISSUE TITLE
key pasting issue - slashes escaped

### ISSUE THREAD

**Issue by gavD** [radify/radiian#57]
I pasted in a key:

```
aaaaaaaaaa/aaaaaaaaaa+aaaaaaaaaaaaaaaaaa
```

My `/ansible/inventory/aws_keys` included:

```
export AWS_SECRET_ACCESS_KEY='aaaaaaaaaa&#x2F;aaaaaaaaaa+aaaaaaaaaaaaaaaaaa'
```

So, the `/` was escaped to `&#x2F;`

**Comment by jfmercer** [comment]
Fixed in #65

**Comment by nateabele** [comment]
Best practice is to close issues once the relevant PR has been merged.
